extendr-api/src/ownership.rs

```diff
@@ -145,52 +228,18 @@ impl Ownership {
             ref mut objects,
         } = *self;
 
-        let sexp_usize = sexp as usize;
+        let sexp_usize = sexp.into();
```

Comment:
I think you can delay this, and just use `.entry(sexp.into())`, no?

Reply:
Oh.. can you explain a little bit?

Reply:
I meant do not declare separate variable and inline the cast. But it is unimportant.